include/deal.II/base/partitioner.h

```diff
@@ -204,6 +204,16 @@ namespace Utilities
        */
       Partitioner(const unsigned int size);
 
+      /**
+       * Constructor that takes the number of locally-owned degrees of freedom
+       * @p and the number of ghost degrees of freedom @p ghost_size.
+       *
+       * @note An interpretation of the ghost indices is not possible.
+       */
+      Partitioner(const unsigned int size,
+                  const unsigned int ghost_size,
+                  const MPI_Comm     communicator);
```

Comment:
I realize this is the same type as in the other constructor, but I guess it would make sense to make these `types::global_dof_index`.